samples/typescript/devconsole.ts/src/index.ts

```diff
@@ -26,61 +26,196 @@ function enumerateNamespaces(topLevelNamespace: any) {
         .filter(n => !exceptions.includes(n));
 }
 
-(async function () {
-    class AddressCoin {
-        coin: any;
-        address: string;
-        constructor(coin: string, address: string) {
-            this.coin = coin;
-            this.address = address;
+async function main() {
+    // Wrapper for a wallet stored in keystore
+    class StoredKeyWallet {
+        wallet: any; // StoredKey
+        constructor(w: any) {
+            this.wallet = w;
         }
-    };
+        status(): string {
+            return `Wallet '${this.wallet.name()}', with ${this.wallet.accountCount()} addresses`;
+        }
+        dump(): void {
+            console.error(`Wallet '${this.wallet.name()}', ${this.wallet.isMnemonic() ? 'type mnemonic' : ''}:`);
 
-    class TestWallet {
-        name: string;
-        wallet: any;
-        addresses: AddressCoin[];
+            if (this.wallet.accountCount() == 0) {
+                console.log('No addresses');
+            } else {
+                console.log('Addresses:');
+                const n = this.wallet.accountCount();
+                for (var i = 0; i < n; ++i) {
+                    const a = this.wallet.account(i);
+                    console.log(`  ${WC.CoinTypeConfiguration.getSymbol(a.coin())}:  ${a.address()}`);
+                }
+            }
+        }
+    }
 
-        constructor(name: string = '', wallet: any) {
-            this.name = name;
-            this.wallet = wallet;
-            this.addresses = [];
-            this.addDefaultCoins();
+    // Handles several wallets, with keystore
+    class TestWalletManager {
+        // single opened StoredKeyWallet instance
+        wallet: StoredKeyWallet | null = null;
+        StoreFolderPath = "/tmp/devconsole.ts";
+        StoreFixedPassword = "devconsole.ts";
+        HDWalletPassord = '';
+        DefaultCoins: any[] = [];
+        keystore: any;
+        // wallets available in storage
+        storeWallets: any[] = [];
+        
+        init(): void {
+            this.DefaultCoins = [WC.CoinType.bitcoin, WC.CoinType.ethereum, WC.CoinType.binance];
+            fs.mkdir(this.StoreFolderPath, () => {});
```

Comment:
let's handle cases when this fails to work

Reply:
Good point, added error handling for Filesystem problems.